### ISSUE TITLE
Database Generation

### ISSUE THREAD

**Issue by aabryant** [TrumpRussiaInvestigationTwitterResearch/trump-russia-bot-analysis#3]
The database generation script relies on a CSV file that cannot be included in the repository. @thomas-keller has indicated that he can supply an alternate CSV file and a new script to format that such that it can be used to generate the database.

**Comment by thomas-keller** [comment]
Hi,

I slightly misunderstand which part of the project this was referring to. I originally planned just to make a csv of the IDs for the final dataset. I guess we might need to think a little more if we want to go all the way make to regenerating the original database of tweets with the full month of data.

**Comment by aabryant** [comment]
I see. Ideally this repository will ultimately contain everything necessary for an end user to clone it and then make full use of the scripts, so we'll need to do some thinking about how to handle that.

**Comment by thomas-keller** [comment]
Yeah, I'm with you generally on having fully reproducible analyses. I pushed the final id csv and the associated code to regenerate from that file. Going back to the database step will take some extra time.

**Comment by aabryant** [comment]
Looks good! Per Dr. Hagen's instructions earlier, I don't think there's any immediate rush to get this all set up, so you can take however much time you need. Be sure to let me know if I can help with anything.
